GrowthBook/src/commonTest/kotlin/com/sdk/growthbook/tests/GBTestCases.kt

```diff
@@ -219,18 +219,6 @@ val gbTestCases = """
           },
           false
         ],
-        [
```

Comment:
Remove the test, because correct expected result is `false`. Check for the case migrates to a isolate test. Because the test with the long JSON does not easy for debug.